package.json

```diff
@@ -200,11 +199,13 @@
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^14.2.1",
     "@types/bignumber.js": "^5.0.0",
+    "@types/bn.js": "^5.1.1",
```

Comment:
Partially revert 53a7a512baed6740e81e56920c5269c77b51f207 - authored https://github.com/shapeshift/web/pull/3414 as a safer way to be type-safe

Reply:
03c5b65f2c7fe8baa366f0859374c91c4fdfae69